indexer/editable_map_object.cpp

```diff
@@ -70,6 +72,99 @@ bool IsProtocolSpecified(string const & website)
 {
   return GetProtocolNameLength(website) > 0;
 }
+osm::FakeNames MakeFakeSource(StringUtf8Multilang const & source,
+                              vector<int8_t> const & mwmLanguages, StringUtf8Multilang & fakeSource)
+{
+  string defaultName;
+  // Fake names works for mono language (official) speaking countries-only.
+  if (source.IsEmpty() || mwmLanguages.size() != 1 ||
```

Comment:
No need to check that source.IsEmpty(). When StringUtf8Multilang is empty, all calls to GetString() return false, right?